relay-event-schema/src/protocol/contexts/mod.rs

```diff
@@ -94,6 +96,8 @@ pub enum Context {
     PerformanceScore(Box<PerformanceScoreContext>),
     /// Spring / Spring Boot information.
     Spring(Box<SpringContext>),
+    /// Expo Updates information.
+    ExpoUpdates(Box<ExpoUpdatesContext>),
```

Comment:
This is very external product specific. 
Are there any contexts we could uses to set those fields? 
Or also have more generic name for this, so we could re-use this `Updates` in the future?